src/components/Settings/Organization/Contacts/ContactRow/ContactRow.tsx

```diff
@@ -235,9 +235,10 @@ export const ContactRow: React.FC<Props> = ({ contact }) => {
         message={
           <Trans
             t={t}
-            defaults="<p>This contact will be anonymized in your {{appName}} organization. This is permanent and can't be recovered. Only anonymize if you are 100% confident that you are looking at the correct contact.</p><br /><p>A contact placeholder will remain with a name like “DataPrivacy, Deleted”. Gift data will remain. Other data such as notes and tasks will be removed. Status will be set as “Never Ask”. Newsletter set to 'N/A'. You can request removal across all other systems at dsar@cru.org.</p>"
+            defaults="<p>This contact will be anonymized in your {{appName}} organization. This is permanent and can't be recovered. Only anonymize if you are 100% confident that you are looking at the correct contact.</p><br /><p>A contact placeholder will remain with a name like “DataPrivacy, Deleted”. Gift data will remain. Other data such as notes and tasks will be removed. Status will be set as “Never Ask”. Newsletter set to 'N/A'. You can request removal across all other systems at {{serviceSupportEmail}}.</p>"
             values={{
               appName,
+              serviceSupportEmail: process.env.SERVICE_SUPPORT_EMAIL,
```

Comment:
I wonder if this name might be confusing to other orgs. Would `DATA_PRIVACY_EMAIL` or something similar be a better name?